drivers/sdhc/Kconfig.mcux_sdif

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022, NXP
+# SPDX -License-Identifier: Apache-2.0
+
+config MCUX_SDIF
+	bool "NXP MCUX SDIF Driver"
+	depends on HAS_MCUX_SDIF
```

Comment:
change this to:

```
default y
depends on DT_HAS_NXP_LPC_SDIF_ENABLED`
```

remove the `depends on HAS_MCUX_SDIF`

Reply:
Fixed now